cmd/server/main.go

```diff
@@ -22,10 +22,11 @@ import (
 	"github.com/Azure/kubernetes-kms/pkg/version"
 
 	"google.golang.org/grpc"
-	pb "k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1"
```

Comment:
In a follow up lets see if we can drop the `k8s.io/apiserver` dep altogether.

Reply:
Actually looks like the only reference left is in `./tests/client/client_test.go` so maybe just fix that now as well?

Reply:
Yup. I just fixed it.